src/darwin/Framework/CHIP/MTRDeviceOverXPC.mm

```diff
@@ -46,7 +46,7 @@ - (instancetype)initWithControllerOverXPC:(MTRDeviceControllerOverXPC *)controll
                                  deviceID:(NSNumber *)deviceID
                             xpcConnection:(MTRDeviceControllerXPCConnection *)xpcConnection
 {
-    _controllerID = controllerOverXPC.controllerID;
+    _controllerID = controllerOverXPC.controllerXPCID;
```

Comment:
Will you be keeping MTRDeviceOverXPC's `controllerID` name?

Reply:
No, it's not colliding with anything for now.